models/issue_reaction.go

```diff
@@ -68,9 +68,18 @@ func FindIssueReactions(issue *Issue) (ReactionList, error) {
 func findReactions(e Engine, opts FindReactionsOptions) ([]*Reaction, error) {
 	reactions := make([]*Reaction, 0, 10)
 	sess := e.Where(opts.toConds())
```

Comment:
This should work:
```
.NotIn("reaction.`type`", setting.UI.ReactionsMap key array)
```

Reply:
@lafriks figure it out - thanks the `.In(...)` hint was it :d - was also writen in discord but had no time until jet to change it

-> DONE

Reply:
@6543 Better use ``` .`type` ``` as @lafriks proposed